app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Let's include a new action for the comments to be created, a new action is what will call the view that contains a form.